docs/manual/commands/shell/qtile-migrate.rst

```diff
@@ -0,0 +1,146 @@
+qtile migrate
+=============
+
+``qtile migrate`` is a tool to help users update their configs to
+reflect any breaking changes/deprecations introduced in later versions.
+
+The tool can automatically apply updates but it can also be used to highlight
+impacted lines, allowing users to update their configs manually.
+
+The tool can take a number of options when running:
+
+.. list-table::
+    :widths: 10 45 45
+    :header-rows: 1
+
+    * - Argument
+      - Description
+      - Default
+    * - ``-c``, ``--config``
+      - Sets the path to the config file
+      - ``~/.config/qtile/config.py``
+    * - ``--list-migrations``
+      - Lists all the available migrations that can be
+        run by the tool.
+      - n/a
+    * - ``--info ID``
+      - Show more detail about the migration implement by
+        ID.
+      - n/a
+    * - ``--after-version VERSION``
+      - Only runs migrations relating to changes implemented
+        after release VERSION.
+      - Not set (i.e. runs all migrations).
+    * - ``-r ID``, ``--run-migrations ID``
+      - Run selected migrations identified by ID. Comma separated
+        list if using multiple values.
+      - Not set (i.e. runs all migrations).
+    * - ``--yes``
+      - Automatically apply changes without asking user for confirmation.
+      - Not set (i.e. users will need to confirm application 
+        of changes).
+    * - ``--show-diff``
+      - When used with ``--yes`` will cause diffs to still be shown for
+        information purposes only.
+      - Not set.
+    * - ``--no-colour``
+      - Disables colour output for diff.
+      - Not set
+    * - ``--lint``
+      - Outputs linting lines showing location of changes.
+        No changes are made to the config.
+      - Not set.
+
+Available migrations
+--------------------
+
+The following migrations are currently available.
```

Comment:
Supposed to be empty?

Reply:
It's not empty. The next lines are this:
```
.. qtile_migrations::
  :summary:
```
That generates a table of the migrations automatically i.e. if we add/remove migrations, the docs will automatically update when they're rebuilt. This way we never need to edit the docs directly.